packages/backend/src/env.ts

```diff
@@ -64,10 +85,14 @@ export class Env {
         this.authStrategy = new AuthStrategy(env.JWT_SIGNING_KEY);
 
         this.anonymousIdDao = new AnonymousIdDao(env.HASHED_IP, env.POLYRATINGS_SESSIONS);
+
+        this.ADD_RATING_LIMITER = env.ADD_RATING_LIMITER;
     }
 }
 
-export type CloudflareEnv = z.infer<typeof cloudflareEnvParser>;
+export type CloudflareEnv = z.infer<typeof cloudflareEnvParser> & {
```

Comment:
I think it would be best to use Zods `z.function` here with a default value. We could define a parser like

```ts
const cloudflareRateLimiter = z
    .function()
    .args(z.object({ key: z.string() }))
    .returns(z.object({ sucsess: z.boolean() }))
    .optional()
    .default(() => ({ sucsess: true }));

type CloudflareRateLimiter = z.type<typeof cloudlfareRateLimitor>

const cloudflareEnvParser = {
...,
ADD_RATING_LIMITER: cloudflareRateLimiter
}
```

Reply:
Unfortunately, it's more complicated than a function. I need it to parse an interface. Regardless I remember having trouble pushing it through Zod. I'll experiment with it

Reply:
revisited this and although it parses through OK, it seems to break the function from working. I'm keeping the existing logic :(